guides/manual-installation.md

```diff
@@ -0,0 +1,279 @@
+## Scope
+
+This documents covers the step to manually install trento server without relying on the trento helm chart or kubernetes all together. The lastest
+available version of SUSE Linux Enterprise Server for SAP Applications is used as the base operating system, which is 15 SP5 at the time of writing.
+
+## List of dependencies
+
+- prometheus
+- postgresql
+- rabbitmq
+- docker runtime
```

Comment:
Maybe add an installation step for installing docker?
Just in case to ensure that the reader has everything